cloud.ks

```diff
@@ -11,118 +10,98 @@ rootpw --lock --iscrypted locked
 # higher level firewall constructs (i.e. security groups).
 firewall --disabled
 
-# console=ttyAMA0 and console=hvc0 as kernel boot parameter to see
-# kernel boot messages on serial console as well on aarch64 and
-# ppc64le respectively.
-# https://pagure.io/atomic-wg/issue/347
-bootloader --timeout=1 --append="no_timer_check console=tty1 console=ttyS0,115200n8 console=ttyAMA0 console=hvc0 net.ifnames=0"
-
-network --bootproto=dhcp --device=link --activate --onboot=on
+network --bootproto=dhcp --onboot=on
 services --enabled=sshd,cloud-init,cloud-init-local,cloud-config,cloud-final
+services --disabled=network,avahi-daemon
 
 zerombr
-clearpart --all
-# Implement: https://pagure.io/atomic-wg/issue/281
-# The bare metal layout default is in http://pkgs.fedoraproject.org/cgit/rpms/fedora-productimg-atomic.git
-# However, the disk size is currently just 6GB for the cloud image (defined in pungi-fedora).  So the
-# "15GB, rest unallocated" model doesn't make sense.  The Vagrant box is 40GB (apparently a number of
-# Vagrant boxes come big and rely on thin provisioning).
-# In both cases, it's simplest to just fill all the disk space.
-#
-# Use reqpart to create hardware platform specific partitions
-# https://pagure.io/atomic-wg/issue/299
-reqpart --add-boot
+clearpart --initlabel --all
+bootloader --timeout=1 --append="no_timer_check console=tty1 console=ttyS0,115200n8 net.ifnames=0 biosdevname=0"
+
+part /boot --size=300 --fstype="xfs"
 part pv.01 --grow
 volgroup atomicos pv.01
```

Comment:
While we have the patient open, `s/atomicos/coreos`?

Reply:
And let's not block on this but @bgilbert and @ajeddeloh noted that Ignition today does not support LVM.